GeosCore/diagnostics_mod.F90

```diff
@@ -936,7 +936,8 @@ SUBROUTINE Compute_Budget_Diagnostics( Input_Opt,   State_Chm, State_Grid, &
              IF ( before ) THEN
                 colMass(I,J,N,1) = colSum
              ELSE
-                diagFull(I,J,S) = ( colSum - colMass(I,J,N,1) ) / timeStep
+                diagFull(I,J,S) = ( colSum - colMass(I,J,N,1) ) / timeStep &
+                                / State_Grid%AREA_M2(I,J)
```

Comment:
The budget diagnostics are not per area in the standard model since I found dividing by area lost precision.

Reply:
Does this still work on a CS grid, where the diagnostics are typically regridded to lat-lon (and often to multiple grids)? If so, I'm fine with not including this update. Otherwise, we might want to use MODEL_GEOS blocks here as well.

Reply:
We don't regrid GCHP diagnostics to lat-lon for output, although users can if they wish. For the budget table that used to be in benchmarks we summed the output CS values globally. Hence dividing by area was an extra step we didn't want. I'll put the divide by area in MODEL_GEOS blocks.